bug: nightly fail

```
=========================== short test summary info ============================
FAILED cairo/tests/ethereum/utils/test_bytes.py::TestBytes::test_Bytes_to_Bytes32 - OverflowError: int too big to convert
Falsifying example: test_Bytes_to_Bytes32(
    self=<cairo.tests.ethereum.utils.test_bytes.TestBytes object at 0x7f00ce61e0e0>,
    cairo_run_py=_run,
    a=b'\xbf\x12\xef\x9e\xd2\xb9B\x9b+#8e/\xa0\xa1\xa4\xe9}s\xf71r\xc95F$\xe13\xc25S\xad\xb7"\x17\xb1',
)

You can reproduce this example by temporarily adding @reproduce_failure('6.124.3', b'AH8AJL8S757SuUKbKyM4ZS+goaTpfXP3MXLJNUYk4TPCNVOttyIXsQ==') as a decorator on your test case
```

--- a/cairo/tests/ethereum/utils/test_bytes.py
+++ b/cairo/tests/ethereum/utils/test_bytes.py
@@ -38,6 +38,6 @@ class TestBytes:
         # between the reversed bytes and the input.
         assert cairo_run("ListBytes4_be_to_bytes", a) == b"".join([b[::-1] for b in a])
 
-    @given(a=...)
-    def test_Bytes_to_Bytes32(self, cairo_run_py, a: Bytes):
-        assert cairo_run_py("Bytes_to_Bytes32", a) == Bytes32(a.ljust(32, b"\x00"))
+    @given(a=st.binary(min_size=0, max_size=32))
+    def test_Bytes_to_Bytes32(self, cairo_run, a: Bytes):
+        assert cairo_run("Bytes_to_Bytes32", a) == Bytes32(a.ljust(32, b"\x00"))

